src/platform/posix/backtrace_posix.c

```diff
@@ -1,12 +1,15 @@
-#include "backtrace.h"
+#include "platform/backtrace.h"
+#include "platform/platform.h"
 
+#include "SDL.h"
```

Comment:
External includes should be after internal includes (`SDL.h` should come after).